### PR TITLE
migrate to capistrano 3

### PR DESCRIPTION
**Pull request by dwradcliffe**
FYI @evanphx @qrush @arthurnn @sferik @indirect @kirs

### CODE REVIEW

**Comment by sferik** [comment]
@arthurnn Now that this is merged, can we revert 316874813216353166f0399f6ab450b61d27b8e7?

File: shipit.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tasks:
+  restart:
+    action: Restart application
+    description: Trigger the restart of both unicorn and background workers
+    steps:
+      - cap $ENVIRONMENT deploy:restart
```
Comment (kirs) [inline shipit.yml]: Is it a task for manual restart?

**Comment by kirs** [comment]
:+1:

File: shipit.yml (hunk 1 above)
Comment (dwradcliffe) [inline shipit.yml]: Yep exactly. It shows up as a button in shipit.

**Comment by arthurnn** [comment]
@sferik thanks for the remind...  https://github.com/rubygems/rubygems.org/commit/832f311e73e8561ab6eb7df632e16a1c16980a09

**Pull request merged**